KubeArmor/Makefile

```diff
@@ -39,7 +39,7 @@ run: build
 run-host-only: build
 	cd $(CRDDIR); kubectl apply -f KubeArmorHostPolicy.yaml
 	cd $(CURDIR); sudo rm -f /tmp/kubearmor.log
-	cd $(CURDIR); sudo -E ./kubearmor -logPath=/tmp/kubearmor.log -enableKubeArmorPolicy=false -enableKubeArmorHostPolicy=true
+	cd $(CURDIR); sudo -E ./kubearmor -logPath=/tmp/kubearmor.log -enableKubeArmorPolicy=false -enableKubeArmorHostPolicy=true -k8s=false
```

Comment:
Cannot set k8s to false in the Makefile ... This will impact regular execution.

Reply:
But this is got Kubearmor run-host-only. When run host-only is invoked via make and also .kube/config is present, then host mode is not expected to work.